ecs-cli/modules/compose/ecs/service.go

```diff
@@ -65,6 +67,27 @@ func (s *Service) LoadContext() error {
 		MaximumPercent:        maxPercent,
 		MinimumHealthyPercent: minHealthyPercent,
 	}
+
+	targetGroupArn := s.Context().CLIContext.String(TargetGroupArnFlag)
+	containerName := s.Context().CLIContext.String(ContainerNameFlag)
+	containerPort, err := getInt64FromCLIContext(s.Context(), ContainerPortFlag)
+	if err != nil {
+		return err
+	}
+	loadBalancerName := s.Context().CLIContext.String(LoadBalancerNameFlag)
+
+	s.loadBalancer = &ecs.LoadBalancer{
+		ContainerName: aws.String(containerName),
+		ContainerPort: containerPort,
+	}
+
+	if targetGroupArn != "" {
```

Comment:
What I did in with the other params is let the AWS API validate the input of the user, the validation errors are quite meaninfull in most of the cases.

If we remove the `targetGoupArn`/`loadBalancerName` fallback here we get a good server side message educating the user: 

`FATA[0000] InvalidParameterException: loadBalancerName and targetGroupArn cannot both be specified. You must specify either a load BalancerName or a targetGroupArn.`

So removing the checking here